cql3/query_processor.cc

```diff
@@ -1041,6 +1041,7 @@ query_processor::execute_schema_statement(const statements::schema_altering_stat
             co_await remote_.get().mm.announce<service::topology_change>(std::move(m), std::move(*guard), description);
             // TODO: eliminate timeout from alter ks statement
             auto error = co_await remote_.get().ss.wait_for_topology_request_completion(request_id);
+            co_await remote_.get().ss.wait_for_topology_not_busy();
```

Comment:
Shouldn't it wait in a loop like `storage_service::transit_tablet()` does?

Reply:
`wait_for_topology_not_busy` is implemented as:
```
future<> storage_service::wait_for_topology_not_busy() {
    while (_topology_state_machine._topology.is_busy()) {
        co_await _topology_state_machine.event.wait();
    }
}
```
and `storage_service::transit_tablet()` has this internal wait loop:
```
        while (_topology_state_machine._topology.is_busy()) {
            const auto tstate = *_topology_state_machine._topology.tstate;
            if (tstate == topology::transition_state::tablet_draining ||
                tstate == topology::transition_state::tablet_migration) {
                break;
            }
            rtlogger.debug("transit_tablet(): topology state machine is busy: {}", tstate);
            release_guard(std::move(guard));
            co_await _topology_state_machine.event.wait();
            guard = co_await _group0->client().start_operation(&_group0_as, raft_timeout{});
        }
```
Should I add anything else to "my" loop to make it wait on something else?

Reply:
Should I adjust the code? Or is it OK?

Reply:
@xemul I added raft guard to the function.